cost_reports/app.py

```diff
@@ -11,6 +11,7 @@
 
 # other imports
 import boto3
+from urllib.parse import unquote as url_unquote
```

Comment:
why not use [unquote_plus](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.unquote_plus)?

Reply:
hm, I probably should. The character in question is an encoded`=` in the S3 key, but I'm not sure how a space would be encoded.